iac-extensions/kubernetes/src/test/java/org/sonar/iac/kubernetes/checks/MemoryLimitCheckTest.java

```diff
@@ -38,18 +39,20 @@ void testPodKind() {
 
   @Test
   void testPodKindWithGlobalLimit() {
-    KubernetesVerifier.verifyNoIssue("MemoryLimitCheck/memory_limit_pod_with_global_limit.yaml", check, "MemoryLimitCheck/limit_range.yaml");
+    KubernetesVerifier.verifyNoIssue("MemoryLimitCheck/memory_limit_pod_with_global_limit.yaml", check, "MemoryLimitCheck/limit_ranges.yaml");
   }
 
-  @Test
-  void testPodKindWithGlobalLimitWrongType() {
-    KubernetesVerifier.verify("MemoryLimitCheck/memory_limit_pod.yaml", check, "MemoryLimitCheck/limit_range_default_namespace.yaml");
-  }
-
-  @Test
-  @DisplayName("Should raise issues when a limit exists in the same namespace (default) but for different resource")
-  void shouldRaiseForPodWithGlobalLimitForCpu() {
-    KubernetesVerifier.verify("MemoryLimitCheck/memory_limit_pod.yaml", check, "MemoryLimitCheck/limit_range_cpu.yaml");
+  @ParameterizedTest
+  @ValueSource(strings = {
+    "with-type-pvc",
+    "with-resource-cpu",
+    "with-fq-type-name",
```

Comment:
What does `fq` mean here? Maybe `with-type-full-qualified-name` or `with-type-fqn` or any other more descriptive name will be better?